doc/api/readline.markdown

```diff
@@ -41,6 +41,9 @@ the following values:
 
  - `historySize` - maximum number of history lines retained. Defaults to `30`.
 
+If a `completer` function is supplied, the user will not be able to enter tab
```

Comment:
How about I just remove it entirely? We can always add it in later if it turns out that people need it explicitly documented.